initrd/bin/kexec-parse-boot

```diff
@@ -205,17 +218,20 @@ syslinux_entry() {
 					state="search"
 					;;
 				*)
-					kernel="${val#"$bootdir"}"
+					kernel="$val"
+					DEBUG "kernel= $kernel"
 			esac
 			;;
 		initrd* | INITRD* )
-			initrd="${val#"$bootdir"}"
+			initrd="$val"
```

Comment:
Will review that part.

Reply:
@JonathonHall-Purism I haven't found a case where this PR caused issue following past comment

Reply:
Will review changes a0753473510ec598f9db86cfb610cf61156ca5cb a0753473510ec598f9db86cfb610cf61156ca5cb and related commits. 

The current issue in master is that when isos are mounted under /boot, the actual bootdir is /boot/boot/ and that is getting stripped otherwise. I think a0753473510ec598f9db86cfb610cf61156ca5cb was relevant until caller scripts fixed it themselves if needed.

Reply:
@tlaurion Sorry this sat in my inbox for a few days.  Makes sense, agree it looks like the root issue was fixed.  I see what you mean that stripping /boot from /boot/boot/... would cause this issue.

Is this ready to merge then?